{"uuid":"eb5bbb1f-78ba-46d8-b6c8-813343529116","historyId":"a38d2e20f1202517dca2b0834af9e40a","fullName":"com.api.crud.TC_CreateBooking.testCreateBooking_Invalid","labels":[{"name":"package","value":"com.api.crud.TC_CreateBooking"},{"name":"testClass","value":"com.api.crud.TC_CreateBooking"},{"name":"testMethod","value":"testCreateBooking_Invalid"},{"name":"parentSuite","value":"All Test Suite"},{"name":"suite","value":"E:/Intellij_Projects/API_Automation/RestfulBooker/src/test"},{"name":"subSuite","value":"com.api.crud.TC_CreateBooking"},{"name":"host","value":"LAPTOP-5RQINP32"},{"name":"thread","value":"14648@LAPTOP-5RQINP32.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"owner","value":"Mahendra"},{"name":"severity","value":"critical"}],"links":[],"name":"testCreateBooking_Invalid","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.lang.ExceptionInInitializerError","trace":"java.lang.ExceptionInInitializerError\r\n\tat com.api.base.BaseTest.setUp(BaseTest.java:33)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:393)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:326)\r\n\tat org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:661)\r\n\tat org.testng.TestRunner.beforeRun(TestRunner.java:650)\r\n\tat org.testng.TestRunner.run(TestRunner.java:612)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:443)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:336)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1228)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1134)\r\n\tat org.testng.TestNG.run(TestNG.java:1101)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\nCaused by: java.lang.RuntimeException: com.codoid.products.exception.FilloException: Invalid Query - SELECT * FROM SHEET1 WHERE ID = BASEURL;\r\n\tat com.api.endpoints.APIConstants.<clinit>(APIConstants.java:16)\r\n\t... 22 more\r\nCaused by: com.codoid.products.exception.FilloException: Invalid Query - SELECT * FROM SHEET1 WHERE ID = BASEURL;\r\n\tat com.codoid.products.parser.SelectQueryParser.initializeQuery(SelectQueryParser.java:46)\r\n\tat com.codoid.products.fillo.Select.getRecordset(Select.java:44)\r\n\tat com.codoid.products.fillo.Connection.executeQuery(Connection.java:64)\r\n\tat com.api.utils.FilloUtil.fetchDataFromXLSX(FilloUtil.java:18)\r\n\tat com.api.endpoints.APIConstants.<clinit>(APIConstants.java:14)\r\n\t... 22 more\r\n"},"stage":"finished","description":"TC#2 - Create booking invalid values in request test case","steps":[],"attachments":[],"parameters":[],"start":1723202107414,"stop":1723202107414}